Magento_Theme/templates/root.phtml

```diff
@@ -1,24 +1,24 @@
 <!doctype html>
-<html <?= $htmlAttributes ?>>
-    <head <?= $headAttributes ?>>
+<html <?= /* @noEscape */ $htmlAttributes ?>>
+    <head <?= /* @noEscape */ $headAttributes ?>>
         <script
             src="<?= $this->getViewFileUrl('Magento_Theme::js/lib/core.min.js'); ?>"
```

Comment:
Here should be `src="<?= $block->escapeUrl($this->getViewFileUrl('Magento_Theme::js/lib/core.min.js')); ?>"` but `$block` doesn't work, Could you check it ?